src/util/clocktime.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use self::inner::now_monotonic_raw_clocktime;
+
+#[cfg(not(any(target_os = "linux")))]
+mod inner {
+    use time::{self, Timespec};
+
+    const NANOSECONDS_PER_SECOND: u64 = 1_000_000_000;
+
+    pub fn now_monotonic_raw_clocktime() -> Timespec {
```

Comment:
can we use a shorter name like `raw_now()` or other? The mod name is `clocktime`, so the function looks redundant with postfix `clocktime`.